### PR TITLE
Reformatting RELU function, added docstring examples

### PR DESCRIPTION
**Pull request by dimikave**

### CODE REVIEW

**Comment by dimikave** [comment]
add_reformatting_checklist_relu

**Comment by dimikave** [comment]
Close #6982

**Comment by dimikave** [comment]
As far as I can see, the docstrings related tests that fail, seem to fail on other functions and not on relu. As for the other tests, I am not sure why the fail or if they fail in the original branch as well. 

Looking forward for your review,
Thanks in advance

File: ivy/functional/ivy/activations.py [hunk 1]
```diff
@@ -95,8 +108,8 @@ def leaky_relu(
     ret
         The input array with leaky relu applied element-wise.
 
-    Functional Examples
-    -------------------
+    Examples
+    --------
```
Comment (Daniel4078) [inline ivy/functional/ivy/activations.py:113]: ivy/functional/ivy/activations.py:94 in public function `leaky_relu`:
        D412: No blank lines allowed between a section header and its content ('Examples')

**Review by Daniel4078 — CHANGES_REQUESTED** [review]
Hello, I only found one small error for you to fix. Thank you.

File: ivy/functional/ivy/activations.py (hunk 1 above)
Comment (dimikave) [reply ivy/functional/ivy/activations.py:113]: Hello @Daniel4078 , as far as I can say, leaky_relu is a different function tracked by someone alse on the specific issue: https://github.com/unifyai/ivy/issues/6626

As I understand I should not interfere with someone else work, is that correct? In any case, if you still want me to fix this error it is pretty easy and minor and of course I'll handle it. Can you please clarify?

Comment (Daniel4078) [reply ivy/functional/ivy/activations.py:113]: Well, I just saw in the change log and find this error is from a place that you changed. Of course you can fix existing errors by others as long as no new critical error are introduced. Thank you!

**Review by Daniel4078 — APPROVED** [review]
Well done!

**Pull request merged**